incubator/cni-genie/values.yaml

```diff
@@ -0,0 +1,62 @@
+# ServiceToken: ""
+
+# By default, complete genie package will be installed including
+# multi networking support (along with admission controller) and network policy controller
+# In case network policy controller is not needed, set 'includePolicy' flag to false
+includePolicy: true
+
+image:
+  pullPolicy: IfNotPresent
+
+  # Paths for fetching genie images
+  pluginImagePath: quay.io/huawei-cni-genie/genie-plugin:latest
+
+  admissionControllerImagePath: quay.io/huawei-cni-genie/genie-admission-controller:latest
+
+  policyControllerImagePath: quay.io/huawei-cni-genie/genie-policy-controller:latest
```

Comment:
For the images can you please follow the format [documented in the review docs](https://github.com/helm/charts/blob/master/REVIEW_GUIDELINES.md#configuration).